refactoring.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
```

Comment:
- [ ] Please remove this line and try to fix the linter error by making changes in your code. Trust me this is super easy :blush: If you would make all your methods inside the class then you won't need to face this error :+1: